.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ defaults: &defaults
   working_directory: /home/circleci/project
   resource_class: medium
   docker:
-    - image: geoknee/statechannels:12.18-solc0.6.12 # solc installed (unecessary since hardhat will re-download solc)
+    - image: geoknee/statechannels:12.18-solc0.7.4 # solc installed (unecessary since hardhat will re-download solc)
```

Comment:
This is harmless but probably unnecessary given our use of hardhat for compilation.